### PR TITLE
Update the OS workload

### PR DESCRIPTION
**Pull request by markdryan**
- Increase the allocated memory to 8MiB
- Get rid of the mounted path
- The workload no longer runs stack.sh
- Messages are printed out explaining what to do next

Signed-off-by: Mark Ryan <mark.d.ryan@intel.com>

### CODE REVIEW

**Comment by markdryan** [comment]
@ganeshmaharaj Could you try this out?  Particularly the piece where you copy your local.conf into the VM.  I couldn't get get this bit to work but I think it was just an issue with my local conf file.

**Comment by markdryan** [comment]
@ganeshmaharaj note stack.sh seems a bit racy to me.  Sometimes I get errors about not being able to lock the apt-get file.

**Comment by ganeshmaharaj** [comment]
@markdryan ohh i have never seen that w.r.t devstack. but also haven't used devstack a lot in the recent past to know if this is a symptom. Let me ask around.

**Comment by markdryan** [comment]
H'mm maybe there's something not quite right with the workload then.  I didn't see it with the old workload, but I can possibly see why the changes in this workload should make any difference.  Maybe they've just updated devstack.

**Comment by ganeshmaharaj** [comment]
Right. Changes might happen on devstack and given that we are always tracking the master branch, occasionally drops in the quality are to be expected. I will try one on my side to see if i can reproduce this,

File: workloads/devstack-xenial.yaml [hunk 1]
```diff
@@ -57,14 +51,43 @@ runcmd:
...
+  - {{message . "    HOST_IP=127.0.0.1" }}
+{{end}}
+  - {{message . "    ADMIN_PASSWORD=secret" }}
+  - {{message . "    DATABASE_PASSWORD=\\$ADMIN_PASSWORD" }}
+  - {{message . "    RABBIT_PASSWORD=\\$ADMIN_PASSWORD" }}
+  - {{message . "    SERVICE_PASSWORD=\\$ADMIN_PASSWORD" }}
+  - {{message . "" }}
+  - {{message . "To start devstack with these defaults type" }}
+  - {{message . "" }}
+  - {{message . (printf "    ccloudvm run %s 'cd devstack && ./stack.sh'" .Hostname) }}
```
Comment (ganeshmaharaj) [inline workloads/devstack-xenial.yaml]: This command is already running on line no. 58. Maybe this should say "if the defaults are of our preference, no action is required and you can follow stack.log in $HOME"

File: workloads/devstack-xenial.yaml [hunk 2]
```diff
@@ -57,14 +51,43 @@ runcmd:
...
     "RABBIT_PASSWORD=\$ADMIN_PASSWORD"
     "SERVICE_PASSWORD=\$ADMIN_PASSWORD"
-    > {{template "DEVSTACKPATH" .}}/local.conf' {{.User}}
+    > /home/{{.User}}/devstack/local.conf' {{.User}}
   - {{endTaskCheck .}}
-{{ end }}
 
   - {{beginTask . "Stacking the system" }}
-  - su -c "cd {{template "DEVSTACKPATH" .}} && nohup ./stack.sh > /home/{{.User}}/stack.log &" {{.User}}
+  - su -c "cd /home/{{.User}}/devstack && nohup ./stack.sh > /home/{{.User}}/stack.log &" {{.User}}
```
Comment (ganeshmaharaj) [inline workloads/devstack-xenial.yaml]: if we do not plan to start stacking by default, we should get rid of this task.

File: workloads/devstack-xenial.yaml [hunk 3]
```diff
@@ -57,14 +51,43 @@ runcmd:
...
+  - {{message . "    SERVICE_PASSWORD=\\$ADMIN_PASSWORD" }}
+  - {{message . "" }}
+  - {{message . "To start devstack with these defaults type" }}
+  - {{message . "" }}
+  - {{message . (printf "    ccloudvm run %s 'cd devstack && ./stack.sh'" .Hostname) }}
+  - {{message . "" }}
+  - {{message . "To start with a custom local.conf"}}
+  - {{message . "" }}
+  - {{message . (printf "    ccloudvm copy %s local.conf devstack/local.conf" .Hostname) }}
+  - {{message . (printf "    ccloudvm run %s 'cd devstack && ./stack.sh'" .Hostname) }}
```
Comment (ganeshmaharaj) [inline workloads/devstack-xenial.yaml]: We might want to ask them to stop the process, run unstack.sh and then stack.sh

File: workloads/devstack-xenial.yaml (hunk 2 above)
Comment (markdryan) [reply workloads/devstack-xenial.yaml]: Fantastic.  So this is why I was getting all of these errors from stack.sh.  I was running two stack.sh at the same time.

Comment (markdryan) [reply workloads/devstack-xenial.yaml]: I thought I'd removed it but obviously I didn't.  I'll get rid of it now.

Comment (ganeshmaharaj) [reply workloads/devstack-xenial.yaml]: Aha!! that explains the race condition. :)

**Comment by markdryan** [comment]
@ganeshmaharaj It's updated and seems to be working better.

**Comment by ganeshmaharaj** [comment]
@markdryan with my custom local.conf it ran for *real    41m53.621s*

**Pull request merged**